kousa/lib/beef/user.ex

```diff
@@ -48,18 +50,19 @@ defmodule Beef.User do
   end
 
   @doc false
-  def changeset(user, attrs) do
+  def changeset(user, _attrs) do
     user
     |> validate_required([:username, :githubId, :avatarUrl])
   end
 
   def edit_changeset(user, attrs) do
     user
-    |> cast(attrs, [:id, :username, :bio, :displayName])
+    |> cast(attrs, [:id, :username, :bio, :displayName, :externalProfileLink])
     |> validate_required([:username, :bio, :displayName])
     |> validate_length(:bio, min: 2, max: 160)
     |> validate_length(:displayName, min: 2, max: 50)
     |> validate_format(:username, ~r/^(\w){4,15}$/)
+    |> validate_format(:externalProfileLink, ~r/^(http|https):\/\/[^ "]{1,255}$/)
```

Comment:
hi @nadirabbas I think the URL edit can't be nil because the validation `{1,255}` can you try to make it `{0,255}`"?

Reply:
oh damn

Reply:
It works @mohammedgad. Well, that was a dumb mistake. Nice catch 🎉